src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx

```diff
@@ -139,16 +137,16 @@ export const BrowserBar = (props: BrowserBarProps) => {
         <dl className={styles.browserInfoRight}>
           {props.actualChrLocation && (
             <dd className={browserRegionClassName}>
-              <label className="show-for-large">Chromosome</label>
+              <div className={`${styles.chrLabel} show-for-large`}>
```

Comment:
Does the `show-for-large` class serve any purpose? It’s not in scss.

Reply:
It used to when we had Foundation. I'll replace it with the new class.